README.md

```diff
@@ -4,12 +4,14 @@
 
 **Seamlessly build, deploy & manage cloud-native applications.**
```

Comment:
Not sure how you forking your code as the changes to main shouldn't show up as diff here. Can you sync v1.5.2 in your fork and then create a branch out of v1.5.2 in your forked repo? Add your commits in this new branch and publish a PR against v1.5.2.